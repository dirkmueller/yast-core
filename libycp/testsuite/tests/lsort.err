Parsed:
----------------------------------------------------------------------
sort (["z", "체", "채", "c", "b", "a"])
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
lsort (["z", "체", "채", "c", "b", "a"])
----------------------------------------------------------------------
