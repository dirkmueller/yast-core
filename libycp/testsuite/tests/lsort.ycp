sort (["z", "체", "채", "c", "b", "a"])
lsort (["z", "체", "채", "c", "b", "a"])
