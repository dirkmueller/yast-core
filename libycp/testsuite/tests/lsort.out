(["a", "b", "c", "z", "채", "체"])
(["a", "채", "b", "c", "체", "z"])
